### DISCUSSION TITLE
Access Uyuni repos without agent

### DISCUSSION THREAD

**Discussion by sorquan** [uyuni-project/uyuni#9042] [Q&A]
Can i access uyuni repos without agent on systems?

**Comment by rjmateus** [comment]
yes. If you set the correct URL path on the system, to point to the repositories you want. You may need to manually deal with the authentication, or disable repository authentication.
But way aren't use registering the machine on uyuni? You can use ssh contact method, which will not install any agent.

**Reply by sorquan** [reply]
I plan to add basic repositories to the virtual machine template; registration occurs after its deployment in a closed cloud, without access there from uyuni

**Reply by sorquan** [reply]
How i can disable repo auth?

**Reply by rjmateus** [reply]
You can change the set the property "java.salt_check_download_tokens" to false on the "rhn.conf" file.
